docs/content/en/latest/references/embed_semantic_search/_index.md

```diff
@@ -0,0 +1,179 @@
+
+// Import required styles
+import "@gooddata/sdk-ui-semantic-search/styles/css/main.css";
+
+const App = () => {
+    return (
+        <div style={{ width: 300 }}>
+            {/* Wrap the search UI in a container of desired size */}
+            <SemanticSearch
+                // Handle item selection
+                onSelect={(item: ISemanticSearchResultItem) => {
+                    console.log(`Selected item: ${item.title}`);
+                    // Handle the selected item, e.g., navigate to the dashboard
+                }}
+                // Optional: Handle search errors
+                onError={(errorMessage: string) => {
+                    console.error(`Search error: ${errorMessage}`);
+                }}
+                // Optional: Filter by object types
+                objectTypes={["dashboard", "metric"]}
+                // Optional: Enable deep search
+                deepSearch={true}
+                // Optional: Set results limit
+                limit={10}
+                // Optional: Custom placeholder
+                placeholder="Search for dashboards, metrics..."
+            />
+        </div>
+    );
+};
+```
+
+### Props
+
+| Name        | Type                                      | Default | Description                                                                             |
+| ----------- | ----------------------------------------- | ------- | --------------------------------------------------------------------------------------- |
+| locale      | string                                    | -       | Specifies the locale for internationalization. Falls back to context if not specified   |
+| backend     | IAnalyticalBackend                        | -       | Backend instance. Falls back to BackendProvider context if not specified                |
+| workspace   | string                                    | -       | Workspace ID. Falls back to WorkspaceProvider context if not specified                  |
+| onSelect    | (item: ISemanticSearchResultItem) => void | -       | Required callback function called when the user selects an item from the search results |
+| onError     | (errorMessage: string) => void            | -       | Optional callback function called when an error occurs during the search                |
+| className   | string                                    | -       | Additional CSS class for the component                                                  |
+| objectTypes | GenAIObjectType[]                         | []      | A list of object types to search for (e.g., "dashboard", "metric", "insight")           |
+| deepSearch  | boolean                                   | false   | Enable deep search to find dashboards by their contents                                 |
+| limit       | number                                    | 10      | Maximum number of search results to return                                              |
```

Comment:
Limit is tricky because it will not always return exactly this amount.

- It may display less if there not enough semantically close terms;
- it may display more if you count with deep search. E.g. if it found a visualization it will show both the visualization and the dashboard it's used on.

We should add some note about this, otherwise we'll get questions..